DASH-IF-IOP.bs.md

```diff
@@ -10,9 +10,1070 @@ Live discussion in #document-authoring on Slack.
+
+Clauses in section only apply to [=representations=] that use simple addressing.
+
+<figure>
+	<img src="Images/Timing/SimpleAddressing.png" />
+	<figcaption>Simple addressing uses a segment template that is combined with approximate first [=media segment=] timing information and an average [=media segment=] duration in order to reference [=media segments=], either by start time or by sequence number.</figcaption>
+</figure>
+
+The `SegmentTemplate@duration` attribute SHALL define the nominal duration of a [=media segment=] in [=timescale units=].
+
+The set of [=segment references=] SHALL consist of the first [=media segment=] starting exactly at the [=period=] start point and all other [=media segments=] following in a consecutive series of equal time spans of `SegmentTemplate@duration` [=timescale units=], ending with a [=media segment=] that ends at or overlaps the [=period=] end time.
+
+The `SegmentTemplate@media` attribute SHALL contain the URL template for referencing [=media segments=], using either the `$Time$` or `$Number$` template variable to uniquely identify [=media segments=]. The `SegmentTemplate@initialization` attribute SHALL contain the URL template for referencing initialization segments.
+
+If using `$Number$` addressing, the number of the first segment reference SHALL be defined by `SegmentTemplate@startNumber` (default value 1).
+
+<div class="example">
+Below is an example of common usage of the simple addressing mode.
+
+The example defines a [=sample timeline=] with a timescale of 1000 units per second, with the [=period=] starting at position 900. The average duration of a [=media segment=] is 4001. [=Media segment=] numbering starts at 800, so the first [=media segment=] is found at the relative URL `video/800.m4s`. The sequence of [=media segments=] continues to the end of the period, which is 900 seconds long, making for a total of 225 defined [=segment references=].
+
+<xmp highlight="xml">
+<MPD xmlns="urn:mpeg:dash:schema:mpd:2011">
+	<Period duration="PT900S">
+		<AdaptationSet>
+			<Representation>
+				<SegmentTemplate timescale="1000" presentationTimeOffset="900"
+						media="video/$Number$.m4s" initialization="video/init.mp4"
+						duration="4001" startNumber="800" />
+			</Representation>
+		</AdaptationSet>
+	</Period>
+</MPD>
+</xmp>
+
+Parts of the MPD structure that are not relevant for this chapter have been omitted - this is not a fully functional MPD file.
+</div>
+
+#### Inaccuracy in media segment timing when using simple addressing #### {#timing-addressing-inaccuracy}
+
+When using [=simple addressing=], the samples contained in a [=media segment=] MAY cover a different time span on the [=sample timeline=] than what is indicated by the nominal timing in the MPD, as long as no constraints defined in this document are violated by this deviation.
```

Comment:
From @haudiobe: There is a very clear and detailed requirement on this in v4.3. Please check 3.2.7.1

Reply:
I do not understand the purpose of this comment.

I agree that there is some text in 3.2.7.1 that has a certain degree of overlap.

Reply:
OK so we need to combine 3.2.7.1 with this text?

Reply:
Yes, this text is not meant to be purely additive. It restates parts of IOP v4.3 in many places, simply with more clarified wording.